specification/network/resource-manager/Microsoft.Network/preview/2024-10-01-preview/networkManagerBillingMeter.json

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
NOTE: Only GET and UPDATE operations are exposed to customers.
UPDATE is supported only one way for any given NetworkManager from SubscriptionLevel to VnetLevel, and NOT viceversa.

Creation and deletion are admin only operation managed by Azure Virtual Network Manager and CANNOT be given via API

Reply:
please describe this in the 'description' parts of your models